### PR TITLE
HL-85 | Add max file size upload validation, enable private blob storage

### PR DESCRIPTION
**Pull request by quyenlq**

### CODE REVIEW

File: backend/benefit/helsinkibenefit/settings.py [hunk 1]
```diff
@@ -274,6 +275,9 @@
 AZURE_ACCOUNT_NAME = env("AZURE_ACCOUNT_NAME")
 AZURE_ACCOUNT_KEY = env("AZURE_ACCOUNT_KEY")
 AZURE_CONTAINER = env("AZURE_CONTAINER")
+AZURE_URL_EXPIRATION_SECS = env("AZURE_URL_EXPIRATION_SECS")  # default 900s
+
+MAX_UPLOAD_SIZE = 13107200  # 10MB
```
Comment (haavikko) [inline backend/benefit/helsinkibenefit/settings.py:280]: 13107200 / (1024*1024) == 12.5?

File: backend/benefit/applications/api/v1/serializers.py [hunk 2]
```diff
@@ -98,6 +99,13 @@ class Meta:
         ]
         read_only_fields = ["created_at"]
 
+    def validate(self, data):
+        if data["attachment_file"].size > MAX_UPLOAD_SIZE:
+            raise serializers.ValidationError(
+                f"Upload file size cannot be greater than {MAX_UPLOAD_SIZE} bytes"
```
Comment (haavikko) [inline backend/benefit/applications/api/v1/serializers.py:105]: Should error messages like these be localized? I see that I've myself been inconsistent with this

**Review by haavikko — COMMENTED** [review]
Looks good! Added a couple of comments. Also, if it's easy to write a testcase for this, a test would be nice?

File: backend/benefit/applications/api/v1/serializers.py (hunk 2 above)
Comment (haavikko) [reply backend/benefit/applications/api/v1/serializers.py:105]: A separate jira issue was created for this yesterday

File: backend/benefit/helsinkibenefit/settings.py (hunk 1 above)
Comment (quyenlq) [reply backend/benefit/helsinkibenefit/settings.py:280]: Didn't remember why I came up with this number 😅 Yes it should be 10485760

**Review by haavikko — APPROVED** [review]
Looks good to me :-)

**Comment by quyenlq** [comment]
> Looks good! Added a couple of comments. Also, if it's easy to write a testcase for this, a test would be nice?

Added a test for this 😋

**Pull request merged**